tools/query_sloop.c

```diff
@@ -1,22 +1,12 @@
-#include <acme.h>
-static inherit AcmeShadow;
-#include AcmeShadowInc
-
-
-int valid_query_sloop( object ob )
-{
-
-}
-
-object query_sloop( object victim )
-{
-    if( find_shadows( victim, "/usr/gon/tools/sloop_shad")
-    {
-        tell_object( THISP, strformat( "Sloop-a-doop!"));)
-    }
-}
-
-object *find_shadows( object host, string progname )
-{
-
-}
+// Query if player has sloop shadow on them
+// Gon
+// 2017 - 04
+
+// Declared Functions
+int query_sloop();
+
+// Find if target has sloop shadow and return 1
+int query_sloop( object victim ) {
```

Comment:
When I'm returning true/false like this, I like to try and name my function is_something() or has_something() to make it obvious it's a boolean function. In your case, something like is_slooped() would be more informative.

I obsess over naming things, so don't fret on this too much. There's actually an [old saying](https://martinfowler.com/bliki/TwoHardThings.html) about computer science and naming things.